packages/design-system/docs/components/tokens/AllTokens.spec.ts

```diff
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import AllTokens from './AllTokens.vue'
 
-const Constructor = Vue.extend(AllTokens)
+const Constructor = (Vue as any).extend(AllTokens)
```

Comment:
Eeeh... it's probably missing from the type because it doesn't exist anymore?!

Reply:
Yep.. but I'm closing this PR again because we found out that we can't simply remove the `babel-jest` dependency (for now) 😬

Reply:
Can you elaborate for later reference?